### PR TITLE
Status check ts doc

### PR DESCRIPTION
**Pull request by michelle-luna**

### CODE REVIEW

File: jekyll/_cci2/workflows-waiting-status.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+description: "Fixing Workflows Waiting for Status in Github"
+categories: [troubleshooting]
+order: 1
+---
+
+If you have implemented workflows on a branch in your GitHub repository, but the status check never completes, there may be  status settings in GitHub that you need to reset. For example, if you choose to protect your branches, you may need to deselect the status keys you expect from CircleCI, as follows:
+
+![Uncheck GitHub Status Keys]({{ site.baseurl }}/assets/img/docs/github_branches_status.png)
+
+Having the `ci/circleci` checkbox enabled may prevent the status from showing as completed in GitHub because CircleCI posts statuses to Github with a key that includes the job by name in chronological order.
```
Comment (KunalJain) [inline jekyll/_cci2/workflows-waiting-status.md]: Not sure if they are in chronological order when you look at your PR.

**Comment by KunalJain** [comment]
One comment otherwise lgtm

File: jekyll/_cci2/workflows-waiting-status.md (hunk 1 above)
Comment (michelle-luna) [inline jekyll/_cci2/workflows-waiting-status.md]: should I just remove 'in chronological order'? Or add something?

Comment (KunalJain) [inline jekyll/_cci2/workflows-waiting-status.md]: Yup, we can remove that.

**Pull request merged**